operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/client/BulkDataClient.java

```diff
@@ -353,6 +353,8 @@ public PollingLocationResponse status(String job) throws Exception {
                  */
                 if (OperationConstants.FAILED_BAD_SOURCE.equals(bulkExportJobExecutionResponse.getExitStatus())) {
                     throw export.buildOperationException("A bad source input was used during a call to $import", IssueType.INVALID);
+                } else if (OperationConstants.NO_SUCH_BUCKET.equals(bulkExportJobExecutionResponse.getExitStatus())) {
+                    throw export.buildOperationException("No such bucket exists for the storageProvider", IssueType.INVALID);
```

Comment:
will this now result in a 400 error?  it seems like the 500 was actually better here because its actually a server config issue, right?

Reply:
Right, let me switch the type.

Reply:
pushed the change.

Reply:
Changed to IssueType.NO_STORE